sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mcps.c

```diff
@@ -175,7 +201,19 @@ void gnrc_lorawan_mcps_process_downlink(gnrc_lorawan_t *mac, uint8_t *psdu,
                                      key);
     }
 
+#if IS_USED(MODULE_GNRC_LORAWAN_1_1)
+    if (_pkt.port) {
+        mac->mcps.afcnt_down = _pkt.fcnt_down;
+    }
+    else {
+        mac->mcps.fcnt_down = _pkt.fcnt_down;
+    }
+
+    mac->mcps.last_fcnt_down = _pkt.fcnt_down;
+#else
     mac->mcps.fcnt_down = _pkt.fcnt_down;
+#endif
```

Comment:
same here. Please avoid #if macros. In places where it's unavoidable (struct members), please use getters/setters with NO-OPs